### DISCUSSION TITLE
building mage on raspberry

### DISCUSSION THREAD

**Discussion by Himan71** [magefile/mage#387] [Q&A]
Hello,
i tried to build mage on a respberry pi but it did not generate a bin file. There is no error code or message.

![mage](https://user-images.githubusercontent.com/41704859/140613231-18864695-65a7-4656-a6fb-8f08e56abf30.png)

I currently have no idea where i have to look, does somebody have a hint for me?

**Comment by thaney071** [comment]
From the output it should have been placed in `/home/pi/go/bin` as mage.

Usually the ~/go/bin directory gets added to the path so that you can just call it from any where.
